duckduckgo/iOS
# configurable home page

Task/Issue URL: 
* Configurable home page - https://app.asana.com/0/392891325557410/1154997199836817
* Fix misaligned icons - https://app.asana.com/0/414709148257752/1144393195163118

Tech Design URL:
CC:

**Description**:

Split out the config for the home page in to two settings:

* New Tab -> Default or Centered Search and Favorites toggle
* Keyboard -> App Launch and New Tab

New installs now get Default layout and Favorites enabled by default.

When there's no favorites the centered search is static/fixed.

Bookmarks always shows favorites.

Also fixes misaligned icons when starting the app in landscape mode.

**Steps to test this PR**:
1. Test migration.  Existing settings should map to the same experience when updating.
1. Fresh install check default settings
1. Add favorites check UI behaves as expected
1. Check keyboard options behave as expected

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
